### PR TITLE
Adding Persian (Support Iranian)

### PR DESCRIPTION
**Pull request by BaseMax**

### CODE REVIEW

**Review by jbampton — CHANGES_REQUESTED** [review]
Please remove the "All contributors" button from the top of all READMES

And remove the "all contributors" section from the bottom of the README

**Comment by BaseMax** [comment]
Done

**Review by jbampton — CHANGES_REQUESTED** [review]
Looks like the "JO" work is now gone.

We still need the JO link fixed

**Comment by BaseMax** [comment]
This PR is related to Persian. I will send another for the arabic file.

**Review by jbampton — CHANGES_REQUESTED** [review]
If you are adding a new Persian README then we need links in all READMES for it.

Thanks

File: README_FA.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+That's all it takes to get started.
+
+I'm sure this could be a new experience with lots of knowledge to gain and share.
+
+به سلامتی 👍
+
+## مشارکت کنندگان ✨
+
+Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
```
Comment (jbampton) [inline README_FA.md:83]:
```suggestion
```

Comment (BaseMax) [reply README_FA.md]: completed. thanks

**Comment by BaseMax** [comment]
با تشکر از این افراد فوق العاده ([emoji key](https://allcontributors.org/docs/en/emoji-key)):

what is that link for?

**Comment by BaseMax** [comment]
Update: deleted

File: README_FA.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- CircleCI
+- TravisCI
+- Shell Script
+- Pre-commit hooks
+- GitHub Marketplace apps
+
+**توجه:**
+
+برای ملحق شدن تجربه زیاد یا مهارت های عالی لازم نیست.
+
```
Comment (jbampton) [inline README_FA.md:72]:
```suggestion
```

**Review by jbampton — CHANGES_REQUESTED** [review]
Language buttons need to be in order alphabetically

File: README_FA.md (hunk 2 above)
Comment (BaseMax) [reply README_FA.md:72]: done mate

**Review by jbampton — CHANGES_REQUESTED** [review]
README_ES.md is in the wrong order

**Review by jbampton — APPROVED** [review]

**Pull request merged**